packages/map-template/src/components/Map/MapboxMap/ViewmodeSwitch/ViewModeSwitch.jsx

```diff
@@ -0,0 +1,80 @@
+import { useEffect, useState } from 'react';
+import { useRecoilState, useRecoilValue } from 'recoil';
+import solutionState from '../../../../atoms/solutionState';
+import { ReactComponent as Light2D } from '../../../../assets/2d-light.svg';
+import { ReactComponent as Dark2D } from '../../../../assets/2d-dark.svg';
+import { ReactComponent as Light3D } from '../../../../assets/3d-light.svg';
+import { ReactComponent as Dark3D } from '../../../../assets/3d-dark.svg';
+import primaryColorState from '../../../../atoms/primaryColorState';
+import mapboxViewModeState from '../../../../atoms/mapboxViewModeState';
+import './ViewModeSwitch.scss';
+import currentPitchSelector from '../../../../selectors/currentPitch';
+
+export const ViewModes = Object.freeze({
```

Comment:
Maybe we could add comment(s) explaining the values.

Reply:
✅